test/e2e/play_kube_test.go

```diff
@@ -4931,4 +4944,19 @@ spec:
 		Expect(strings.Count(kube.OutputToString(), "Pod:")).To(Equal(1))
 		Expect(strings.Count(kube.OutputToString(), "Container:")).To(Equal(1))
 	})
+
+	It("podman play kube test with hostPID", func() {
+		err := writeYaml(podWithHostPIDDefined, kubeYaml)
+		Expect(err).ToNot(HaveOccurred())
+
+		kube := podmanTest.Podman([]string{"play", "kube", kubeYaml})
+		kube.WaitWithDefaultTimeout()
+		Expect(kube).Should(Exit(0))
+
+		logs := podmanTest.Podman([]string{"pod", "logs", "-c", "test-hostpid-alpine", "test-hostpid"})
```

Comment:
We should also inspect the pod/container to make sure that the setting took the desired effect.